src/com/hms/flexyosisoftconnector/DataPoster.java

```diff
@@ -7,14 +7,40 @@
    private static final int maxDataPointsPerPost = 200;
    private boolean shouldRun;
 
-   public void run()
+   public DataPoster (int communicationType) {
+	   switch(communicationType) {
+       case OSIsoftConfig.omf:
+    	   Logger.LOG_INFO("OMF support enabled");    	   
+           break;
+       case OSIsoftConfig.piwebapi:
+    	   Logger.LOG_INFO("Using connection settings for piwebapi 2018 and earlier");
+           break;
+       default:
+           Logger.LOG_ERR("Immpossible to reach here, checks done in config class.");
+           break;
+       }
+   }
+
+   public void run(int communicationType)

```

Comment:
I need to move this parameter to the constructor so it runs. I'll do that after your review.